### DISCUSSION TITLE
Thumbnail image quality?

### DISCUSSION THREAD

**Discussion by ampossardt** [gilbarbara/react-spotify-web-playback#92] [General]
First off, thanks for making such a great react wrapper for the web playback SDK.

To explain my situation, I'm using this as an easy wrapper to load/work with the player, but I'm hiding the player itself and rendering track info/controls myself. What I've noticed is the current track info contains the small version of the thumbnail which ends up looking blurry when I render it (since my container is much larger than 64x64). I know this isn't anything you control, just curious if you might know about another way I could find the large version of the album cover. I am maintaining a list of albums from the user's library (including the large thumbnail URI), the only problem with this is if songs start playing that aren't part of their library. I can always reach out to the API to pull that info but I'm hoping I'd be able to do just use the info spotify already gives me back.

**Comment by mewil** [comment]
I'm also looking to use a higher quality thumbnail for the album image.

I see the player currently selects the smallest image by width [here](https://github.com/gilbarbara/react-spotify-web-playback/blob/5d785f04159e333b01ae540eb12578637f87aac3/src/index.tsx#L619-L625):
```js
  private setAlbumImage = (album: WebPlaybackAlbum): string => {
    const width = Math.min(...album.images.map(d => d.width));
    const thumb: WebPlaybackImage =
      album.images.find(d => d.width === width) || ({} as WebPlaybackImage);

    return thumb.url;
  };
```
Is there a preferred way to make this configurable as a prop? A prop could simply toggle `max` vs `min`, however some albums have 3 images, so a boolean prop could not select the middle image.

**Comment by gilbarbara** [comment]
@ampossardt @mewil 

I changed the approach in v0.13.0, so the player always uses the large version.
